### DISCUSSION TITLE
Will Xbox be supported as a target?

### DISCUSSION THREAD

**Discussion by BurkusCat** [dotnet/maui#330] [General]
I'm sure this a very niche thing and it is not something I currently do in my apps but will Maui support Xbox (and Hololens) as a target? I noticed that it looks like things are moving from UWP to WinUI 3. Does that mean Xbox/Hololens will no longer be supported targets?

**Comment by Laftek** [comment]
good question!

**Comment by DanJBower** [comment]
It would be really good to see support for both these targets

**Comment by saint4eva** [comment]
> I'm sure this a very niche thing and it is not something I currently do in my apps but will Maui support Xbox (and Hololens) as a target? I noticed that it looks like things are moving from UWP to WinUI 3. Does that mean Xbox/Hololens will no longer be supported targets?

Maui targets Windows based platforms and devices using WinUI 3. So, if WinUI 3 does not support HoloLens and XBox, then Maui could not do otherwise.

**Reply by SliemBeji-FBC** [reply]
Sadly thats true so its unlikly to get support for Hololens and Xbox

**Comment by Strypper** [comment]
LMFAO they swap to WinUI 3 so no
